content/doc/developer/plugin-development/dependency-management.adoc

```diff
@@ -8,14 +8,23 @@ references:
   title: Jenkins Core BOM
 ---
 
-This section details some ways to simplify the management of dependency versions.
+Plugins depend on specific versions of libraries and of other plugins, each of which is versioned independently.
+Each dependency may have dependencies of its own and different versions of different libraries and plugins may not be compatible with each other.
+Moreover, the dependencies can change with each Jenkins release, so managing these dependencies manually can be difficult and time consuming.
 
-== Jenkins Core BOM
+Jenkins has tools that simplify dependency management:
 
-Since version 2.195 (and LTS version 2.164.x) Jenkins Core provides a  link:https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies[Maven Bill Of Materials (BOM)] that centrally defines versions of various libraries used by Jenkins.
+* <<jenkins-core-bom,Jenkins core Bill of Materials (BOM)>> centrally defines the **library versions** provided by Jenkins core
+* <<jenkins-plugin-bom,Plugin Bill of Materials (BOM)>> centrally defines a set of **plugin versions** that have been tested together
+* <<Dependabot>> automatically proposes a pull request when a new version of a dependency is released
+
+== Jenkins core BOM
+
+Jenkins core provides a link:https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Importing_Dependencies[Maven Bill Of Materials (BOM)] that centrally defines versions of various libraries used by Jenkins.
 If you are using Maven to build your plugin, then you can simplify dependency management by importing this BOM, at `jenkins.version`.
 Then, dependency versions will automatically be synchronised with whatever version of Jenkins you are building against.
```

Comment:
Would this section flow better if the information about using the core BOM were after the sample error messages? This info could be unified with what is currently in line 53 ("In many cases, upper bounds dependencies errors are best resolved...")

Reply:
I think that's a good idea.  Can you make that change?